src/platform/site-wide/va-footer/components/CrisisPanel.jsx

```diff
@@ -68,7 +71,10 @@ export default function CrisisPanel() {
                 href="tel:18007994889"
                 aria-label="TTY. 8 0 0. 7 9 9. 4 8 8 9."
               >
-                Call TTY if you have hearing loss <strong>800-799-4889</strong>
+                Call TTY if you have hearing loss{' '}
+                <strong>
+                  <Telephone contact={CONTACTS.SUICIDE_PREVENTION_LIFELINE} />
+                </strong>
```

Comment:
@kelsonic I think we have a link within a link here, producing this error

![image](https://user-images.githubusercontent.com/1915775/89458404-fb250500-d734-11ea-9237-be223f693cfe.png)